_neural-search-plugin/index.md

```diff
@@ -34,7 +34,7 @@ Use `pipeline_name` to create a name for your Neural Search pipeline.
 Field | Data type | Description
 :--- | :--- | :--- 
 description | string | A description of the processor.
-model_id | string | The ID of the model that will be used in the embedding interface. The model must be indexed in OpenSearch before it can be used in Neural Search. For more information, see [Model Serving Framework]
```

Comment:
Why remove this "For more information, see [Model Serving Framework]" ? 

Some user asked about this part. They don't know how to get the model id, I think we should keep the sentence and add link to model serving framework.